activerecord/lib/active_record/attributes.rb

```diff
@@ -260,6 +260,12 @@ def define_attribute(
         define_default_attribute(name, default, cast_type, from_user: user_provided_default)
       end
 
+
+      # TODO(BF): add this to the included
+      # after_load_schema do
+      #   define_attributes_after_schema_loads
+      # end
+      # def define_attributes_after_schema_loads # :nodoc:
```

Comment:
Not sure what's wrong with how the block scope is being evaluated here. Am wondering if I should just forget blocks for now and figure out if we just want to promote a pattern of calling `load_schema!` with super after it.